pages/admin/taskers.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react'
+import { connect } from 'unistore/react'
+
+import Router from '../../lib/router'
+import { actions } from '../../lib/store'
+import { isAdmin } from '../../lib/utils/roles'
+import NotLoggedIn from '../../components/NotLoggedIn'
+import AdminHeader from '../../components/AdminHeader'
+import Link from '../../components/Link'
+
+export class AdminTaskers extends Component {
+  constructor () {
+    super()
+
+    this.state = {
+      loading: true
+    }
+  }
+
+  async componentDidMount () {
+    try {
+      await this.props.getAuthenticatedUser()
+      await this.props.getTaskers()
+      this.setState({ loading: false })
+    } catch (err) {
+      console.error(err)
+      this.setState({ loading: false })
+    }
+  }
+
+  onTMClick (tm) {
+    Router.push(`/admin/taskers/${tm.id}`)
+  }
+
+  renderList () {
+    const { taskers } = this.props
+
+    if (!taskers || !taskers.length) return
+
+    return (
+      <div>
+        <h1>List</h1>
+        <table className='admin-table'>
+          <thead>
+            <tr>
+              <th>Name</th>
+            </tr>
+          </thead>
+          <tbody>
+            {
+              taskers
+                .map((tasker) => (
+                  <tr key={`tasker-${tasker.id}`} onClick={() => this.onTMClick(tasker)} className='admin-table-row'>
+                    <td>{tasker.name}</td>
+                  </tr>
+                ))
+            }
+          </tbody>
+        </table>
+      </div>
+    )
+  }
+
+  render () {
+    const { authenticatedUser } = this.props
+
+    if (this.state.loading) {
+      return (
+        <div><AdminHeader /></div>
+      )
+    }
+
+    if (!authenticatedUser.loggedIn) {
+      return <NotLoggedIn />
+    }
+
+    if (!isAdmin(authenticatedUser.account.roles)) {
+      return Router.push('/')
+    }
+
+    return (
+      <div className='admin'>
+        <AdminHeader />
+        <section>
+          <div className='row'>
+            <div className='sidebar'>
+              <h2 className='header--large'>Tasking Managers</h2>
+              <ul className='admin-sidebar-links'>
+                <li>
+                  <Link href='/admin/taskers/add'>
+                    <a className='link--large'>
+                    Add new Source for Tasks
```

Comment:
I'd expect this to be

```suggestion
                    Add new tasking manager
```